src/tokens.js

```diff
@@ -9,7 +9,7 @@ import buffer from 'buffer';
 import { crypto, util } from 'bitcore-lib';
 import transaction from './transaction';
 import wallet from './wallet';
-import storage from './storage';
+import StorageProxy from './storage_proxy';
```

Comment:
I think the name `StorageProxy` is misleading because it seems we are dealing with a class. I think we should rename it to `storageProxy` in all imports because we are dealing with an instance.

Reply:
Done